lib/tasks/import.rake

```diff
@@ -32,4 +32,12 @@ namespace :import do
 
     puts whitehall_migration_url(whitehall_import.whitehall_migration_id, host: Plek.new.external_url_for("content-publisher"))
   end
+
+  desc "Import an array of documents from Whitehall Publisher using Whitehall's internal document ID e.g. import:whitehall_documents[\"123 456 789\"]"
+  task :whitehall_documents, [:document_ids] => :environment do |_, args|
+    args.document_ids.split(" ").map(&:to_i).each do |document_id|
+      Rake::Task["import:whitehall_document"].reenable
+      Rake::Task["import:whitehall_document"].invoke(document_id)
```

Comment:
I think instead of invoking the single document rake task for each document ID, which creates a `WhitehallMigration` for each document, I think it would be helpful to create just one `WhitehallMigration` and associate that with each `DocumentImport`.  By doing this, we can quickly and easily access all document imports for a migration, rather than having to access each one individually.

Reply:
> I think instead of invoking the single document rake task for each document ID, which creates a `WhitehallMigration` for each document, I think it would be helpful to create just one `WhitehallMigration` and associate that with each `DocumentImport`. By doing this, we can quickly and easily access all document imports for a migration, rather than having to access each one individually.

I like the idea of this, but doesn't it imply that either all documents share the same organisation and/or document_type - which is not necessarily the case as the document id's can be chosen at random by the user, or we'd need to concat these values into fields that are only intended to hold a single value (e.g. "news_story | press_release")?

Reply:
I like the idea, but I do have one reservation with this approach - both `organisation_content_id` and `document_type` will remain `nil`.  

I totally get it that `WhitehallMigration` is being used as a kind of 'Composite', but perhaps if `organisation_content_id` and `document_type` were arrays and populated during the document loop I'd feel happier.  However, that does seem like quite a bit of work for little (if any) value given that we aren't really using those fields for anything meaningful - except for organisation/type/subtype(s) imports where they describe the input params to the rake task - and it doesn't change the UI at all!

Reply:
It's not necessary to populate `organisation_content_id` and `document_type` for `WhitehallMigration` - we don't do that when running the migration for a single document either.  I feel the value comes in the UI where we can get a link to one migration, and can monitor all the documents being imported in one place, rather than having to look at a bunch of separate migration pages.  Similarly, we can access all the document imports from one migration in the console, rather than having to look them up individually.

Reply:
@emmabeynon - Great thanks.  Please see code changes addressing the requested change.